docs/community_resources/faq.md

```diff
@@ -0,0 +1,90 @@
+``` bash
+composer show ezsystems/*
+```
+
+Besides that all the configuration from `app/config` directory might be helpful. It is good to give steps to reproduce the issue, or at least some meaningful description of circumstances under which the problem occurred.
+
+If you stumbled upon some database related problem providing corresponding logs should also be important step.
+
+Also, please remember to mention recent changes, performed migrations or external scripts/code customizations related to the code which generates problem.
+
+#### What are the recommended ways to increase my project's performance?
+
+The most important clues around increasing overall performance of your eZ Platform based project can be found in dedicated [documentation page](../guide/performance.md).
+
+#### How can I translate my admin panel?
+
+The language of the admin panel is based on the browser language. In order to change it you should install proper package for you language, refer to the following [list](https://github.com/ezplatform-i18n). 
+Once you have language packages installed, you can switch the language of the Back Office in the User Settings menu.
+
+If you do not have a language defined in the browser, the language will be selected based on `parameters.locale_fallback` parameter located in `default_parameters.yml`.
+
+If you want to read more about languages managing in eZ Platform please see the following doc pages:
+
+- https://doc.ezplatform.com/en/latest/guide/internationalization/#back-office-languages
+- https://doc.ezplatform.com/en/latest/cookbook/setting_up_multi_language_siteaccesses/
+
+#### How should I proceed with contributions to eZ Platform?
+
+First, create an issue in our [issue tracker](https://jira.ez.no/browse/EZP) and refer to it in commits and pull request headers. For example: `EZP-20104: ContentController should return error status when content is not found`. Then, you can fork our repository and create your own branch from the version of the package which you want to contribute to. 
+
+Now you can introduce whatever changes you wish, either modifying existing files or creating new ones. Once you are done with your edits, add your files to the staging area, then commit with a short, clear description and push changes to your fork. Finally, you can go to the project's page on GitHub and you should see a `Compare and pull request` button. Activate it, write a description and select `Create pull request`. Start the pull request's name with the issue number. Now you can wait for your changes to be reviewed and eventually tested and merged into the product.
+
+Here you can find more details regarding contribution procedure (provided resources apply to different repositories, but the whole process is more or less the same):
```

Comment:
```suggestion
Here you can find more details about the contribution procedure (provided resources apply to different repositories, but the whole process is more or less the same):
```
Maybe also add something like: "Not all pull requests can be accepted, and some will need changes before they can be accepted. When you respond to questions and discussion around your PR, and make changes to it as needed, you increase the chance that it will be accepted, and reduce the waiting time" yaddi yadda etc.